packages/@uppy/angular/angular.json

```diff
@@ -33,5 +33,8 @@
         }
       }
     }
+  },
+  "cli": {
+    "analytics": false
```

Comment:
That looks unrelated
```suggestion
```

Reply:
Yes, but cli keeps asking me this every time I run build locally.